UnityMcpBridge/UnityMcpServer~/src/tools/manage_script.py

```
@@ -306,6 +306,38 @@ def _le(a: tuple[int, int], b: tuple[int, int]) -> bool:
+            if resp.get("success") and (options or {}).get("force_sentinel_reload"):
+                # Optional: flip sentinel via menu if explicitly requested
+                try:
+                    import threading, time, json, glob, os
+                    def _latest_status() -> dict | None:
+                        try:
+                            files = sorted(glob.glob(os.path.expanduser("~/.unity-mcp/unity-mcp-status-*.json")), key=os.path.getmtime, reverse=True)
+                            if not files:
+                                return None
+                            with open(files[0], "r") as f:
+                                return json.loads(f.read())
+                        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)